diff --git a/src/lib/listings.ts b/src/lib/listings.ts
index edfdd1c..c7fabfd 100644
--- a/src/lib/listings.ts
+++ b/src/lib/listings.ts
@@ -27,7 +27,9 @@ export const SUPPORTED_CITIES = [
   'Вологда',
   'Воронеж',
   'Грозный',
+  'Джанкой',
   'Дзержинск',
+  'Евпатория',
   'Екатеринбург',
   'Иваново',
   'Ижевск',
@@ -37,6 +39,7 @@ export const SUPPORTED_CITIES = [
   'Калининград',
   'Калуга',
   'Кемерово',
+  'Керчь',
   'Киров',
   'Кострома',
   'Краснодар',
@@ -68,6 +71,8 @@ export const SUPPORTED_CITIES = [
   'Самара',
   'Саранск',
   'Саратов',
+  'Севастополь',
+  'Симферополь',
   'Смоленск',
   'Сочи',
   'Ставрополь',
@@ -84,6 +89,7 @@ export const SUPPORTED_CITIES = [
   'Улан-Удэ',
   'Ульяновск',
   'Уфа',
+  'Феодосия',
   'Хабаровск',
   'Чебоксары',
   'Челябинск',
@@ -92,6 +98,7 @@ export const SUPPORTED_CITIES = [
   'Элиста',
   'Южно-Сахалинск',
   'Якутск',
+  'Ялта',
   'Ярославль',
 ] as const;
 
